### DISCUSSION TITLE
How to enable Renovate on new repo when I've selected specific repos on intall

### DISCUSSION THREAD

**Discussion by mbelangergit** [renovatebot/renovate#21225] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

I've started to use Renovate in my GitHub account on one of my repository, `repo1`. So far so good.

When I've installed the app, I've selected the "Only select repositories" and enabled Renovate only on `repo1`.
![image](https://user-images.githubusercontent.com/56379077/228543807-2c6b93b8-7629-4e75-bf0e-849c2db31400.png)

Now I want to add Renovate on another repository, `repo2`. I'm a bit confused, I can't find anymore the options above to re-authorize Renovate for this new repo `repo2`.

Here what I see in `repo2` GitHub apps settings category:
![image](https://user-images.githubusercontent.com/56379077/228541885-6e6b8dc2-06b2-406d-be58-6611f65c0fd2.png)

And here is what I see in my account GitHub Applications settings category:
![image](https://user-images.githubusercontent.com/56379077/228542274-c350cd78-133b-44df-9f0a-f0f7abfbd79f.png)
![image](https://user-images.githubusercontent.com/56379077/228542356-c95e78ab-b52a-4457-a8d6-94ce8a4c5d7b.png)

When I click on Renovate in Authorized GitHub Apps, I only see a `Revoke access` button
![image](https://user-images.githubusercontent.com/56379077/228542703-007376e2-deac-45cc-a15e-b7a22cc4c0b2.png)

And when click on `Configure access` button in the Renovate page in GitHub Marketplace, it bring me back my account GitHub Applications settings category where as above, I can't do anything except revoking Renovate completely.
![image](https://user-images.githubusercontent.com/56379077/228544183-4a1ca3bc-0dce-442c-ac40-bc7f279ff031.png)

Any idea?

Thanks!

### Relevant debug logs.

N/A

**Comment by rarkins** [comment, marked as answer]
1. Go to https://github.com/apps/renovate and select the "Configure" button
2. Select the right account, authenticate if necessary
3. You'll then be on a page like https://github.com/settings/installations/12345678
4. Scroll down to the bottom!
5. You can select more repositories there

**Reply by mbelangergit** [reply]
Fantastic!
Thanks!
